apps/backoffice-v2/src/common/utils/string-to-rgb/string-to-rgb.ts

```diff
@@ -1,4 +1,4 @@
-const hashString = (str: string) => {
+export const hashString = (str: string) => {
```

Comment:
Suggestion: Move this function to a separated file. Make it a standalone util function.